optimade/server/main.py

```diff
@@ -54,104 +45,51 @@
 
 client = MongoClient()
 structures = MongoCollection(
-    client[CONFIG.mongo_database][CONFIG.mongo_collection], StructureResource
+    client[CONFIG.mongo_database]["structures"], StructureResource, StructureMapper
 )
-
-test_structures_path = (
-    Path(__file__).resolve().parent.joinpath("tests/test_structures.json")
+references = MongoCollection(
+    client[CONFIG.mongo_database]["references"], ReferenceResource, ReferenceMapper
```

Comment:
I think it might be nice to allow the collection names to be user configurable in this example, to allow for people storing the both endpoints under the same mongo collection

Reply:
For sure. I will leave it up to you to come up with the solution for this that satisfies both cases.
Please either commit to this branch, or perhaps add this in another PR, maybe even the one with relationships?